modules/installation-aws-marketplace.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-account.adoc
+
+:_content-type: CONCEPT
+[id="installation-aws-marketplace_{context}"]
+= Supported AWS Marketplace regions
+
+Installing an {product-title} cluster using an AWS Marketplace image is available to customers who purchase the offer in North America.
+
+While the offer must be purchased in North America, you can deploy the cluster to any of the following supported partitions:
+
+* Public
+* GovCloud
+
+[NOTE]
+====
+Deploying an {product-title} cluster using an AWS Marketplace image is not supported for the AWS secret regions or China regions.
```

Comment:
OCP does not support China regions in 4.8, so we can remove it `the AWS secret regions or China regions` -> `the AWS secret regions`

Reply:
Thanks for pointing that out.